packages/widgets/src/lib/components/InferenceWidget/InferenceWidget.svelte

```diff
@@ -28,6 +28,7 @@
 	import ZeroShotImageClassificationWidget from "./widgets/ZeroShotImageClassificationWidget/ZeroShotImageClassificationWidget.svelte";
 	import type { WidgetType } from "@huggingface/tasks";
 	import WidgetInfo from "./shared/WidgetInfo/WidgetInfo.svelte";
+	import { isLoggedIn as isLoggedInStore } from "./stores.js";
```

Comment:
there is a tooling issue that should be resolved in separate PR where js/ts files gets imported with extensions `.js` 
cc: @coyotte508